node/actors/network/src/noise/stream.rs

```diff
@@ -48,7 +48,7 @@ const MAX_PAYLOAD_LEN: usize = MAX_TRANSPORT_MSG_LEN - AUTHDATA_LEN;
 /// <frame len:u16> ++ <frame data:[u8;len]>.
 ///
 /// Length of the frame len field.
-const LENGTH_FIELD_LEN: usize = std::mem::size_of::<u16>();
```

Comment:
It looks like `zksync-era` doesn't like this: https://github.com/matter-labs/zksync-era/actions/runs/10249428261/job/28352782628?pr=2583